Sources/Instrumentation/Tracing/TracingInstrument.swift

```diff
@@ -25,7 +25,12 @@ public protocol TracingInstrument: Instrument {
     ///   - operationName: The name of the operation being traced. This may be a handler function, database call, ...
     ///   - context: The `BaggageContext` within to start the new `Span`.
     ///   - timestamp: The `DispatchTime` at which to start the new `Span`.
```

Comment:
docs for the kind and mentionit defaults to .internal

Reply:
Good catch 👍 Although I wouldn't mention the default up here. It's documented in the protocol extension where the actual default value is being set.

Reply:
Hmm... in jazzy that's a bit hard to discover AFAIR but maybe you're right -- let's just add the param info

Reply:
I'll check how it gets rendered by [swift-doc](https://github.com/swiftdocorg/swift-doc).

Reply:
It could also be worth considering adding something like `swift-doc` to the CI to ensure complete documentation to catch things like this earlier. Do you think that's worth it?

Reply:
would swift-doc fail on it? That's nice but it can also be annoying to have an overly zealous CI about doc comments which we may have in flux hm.

Your call if you want to include it.

I'm ok with code review being the guard for good docs 😉

Reply:
swift-doc can report the coverage of documentation for public API. I'm also happy with leaving it up to code reviews. 👍